### PR TITLE
Make bulleted list for exercises in first two Urbytes

### PR DESCRIPTION
**Pull request by cxk280**

### CODE REVIEW

File: docs/byte/1.md [hunk 1]
```diff
@@ -344,26 +344,24 @@ $
 
 ## Questions and/or exercises
 
-As always, these questions are optional and only for fun.
```
Comment (Fang-) [inline docs/byte/1.md]: I'd probably leave this bit in. People shouldn't feel like they're missing out if they can't/don't want to take the time for these.

**Comment by Fang-** [comment]
Looks good! Two nitpicky things though. First, the comment I left above. Second, there's a paragraph above the bullet list of questions:

> Here's a question to think about.

Now that it's not a """single""" question anymore, that should be rephrased.

Aside from those points, I'm happy to merge these changes in. (:

File: docs/byte/1.md (hunk 1 above)
Comment (cxk280) [inline docs/byte/1.md]: I'll put it back. Please note that I still disagree, though, as I don't really see the questions scaring someone away who read the entire page, and any opportunity to trim down the number of words will probably be useful.

**Comment by cxk280** [comment]
Both points are resolved.

**Comment by Fang-** [comment]
Cool, thanks again!

**Pull request merged**